src/Bootstrap/src/Autoconfig/WebHostBuilderExtensions.cs

```diff
@@ -205,7 +205,7 @@ private static IWebHostBuilder Log(this IWebHostBuilder host, string message)
         private static void WireKubernetesConfiguration(this IWebHostBuilder hostBuilder) =>
             hostBuilder
                 .ConfigureAppConfiguration(cfg => cfg.AddKubernetes(loggerFactory: _loggerFactory))
-                .ConfigureServices(serviceCollection => serviceCollection.AddKubernetesApplicationInstanceInfo())
```

Comment:
This change is probably also needed in `HostBuilderExtenions.cs`

Reply:
Yes I made the same change there as well ...

Reply:
I was referring to [this one](https://github.com/SteeltoeOSS/Steeltoe/blob/main/src/Bootstrap/src/Autoconfig/HostBuilderExtensions.cs)

Reply:
done.